pkg/component/etcd/etcd/etcd_test.go

```diff
@@ -23,13 +23,11 @@ import (
 	"go.uber.org/mock/gomock"
 	appsv1 "k8s.io/api/apps/v1"
 	autoscalingv1 "k8s.io/api/autoscaling/v1"
-	autoscalingv2beta1 "k8s.io/api/autoscaling/v2beta1"
```

Comment:
As the component no longer relies on feature gates, we should be able to clean up https://github.com/gardener/gardener/blob/41a694b024144465bd4f02f39d6d1c3493549c2f/pkg/component/etcd/etcd/etcd_suite_test.go#L19.